app/airq/commands/quality.py

```diff
@@ -100,14 +113,17 @@ def _get_message(self, zipcode: Zipcode) -> typing.List[str]:
         num_desired = 3
         recommended_zipcodes = zipcode.get_recommendations(num_desired)
         if recommended_zipcodes:
-            message.append("Here are the closest places with better air quality:")
+            message.append(
+                gettext("Here are the closest places with better air quality:")
+            )
             for recommendation in recommended_zipcodes:
                 message.append(
-                    " - {} {}: {} ({} mi)".format(
-                        recommendation.city.name,
-                        recommendation.zipcode,
-                        recommendation.pm25_level.display.upper(),
-                        round(
+                    gettext(
+                        " - %(city)s %(zipcode)s: %(pm25_level)s (%(distance)f mi)",
+                        city=recommendation.city.name,
+                        zipcode=recommendation.zipcode,
+                        pm25_level=recommendation.pm25_level.display.upper(),
+                        distance=round(
                             kilometers_to_miles(recommendation.distance(zipcode)),
                             ndigits=1,
                         ),  # TODO: Make this based on locale
```

Comment:
Here I should, do km or miles depending on the locale. I'll look into that

Reply:
Eh, not urgent. If you want to though!